pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -220,6 +219,14 @@ func MakePod(build *v1alpha1.Build, kubeclient kubernetes.Interface) (*corev1.Po
 	}
 	annotations["sidecar.istio.io/inject"] = "false"
 
+	labels := map[string]string{}
```

Comment:
nit: Similar to pipelinerun pattern of adding label this could be similar to `make(map[string]string, len(build.ObjectMeta.Labels)+1)`